en/restore-from-aws-s3-using-br.md

```diff
@@ -34,13 +26,11 @@ Before you restore data from Amazon S3 storage, you need to grant AWS account pe
     kubectl apply -f backup-rbac.yaml -n test2
     ```
 
-2. Create the `s3-secret` secret which stores the credential used to access the S3-compatible storage:
+2. Grant permissions to remote storage.
 
-    {{< copyable "shell-regular" >}}
+    If you restore data from Amazon S3, you can grant permissions by three methods. To grant permissions to access S3-compatible remote storage, refer to [AWS account permissions](grant-permissions-to-remote-storage.md#aws-account-permissions).
```

Comment:
ditto. I don't think "you can grant permissions by three methods." is necessary.